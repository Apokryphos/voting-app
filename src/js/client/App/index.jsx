import React from 'react';
import Title from '../Components/Title';

function App() {
  return <Title text="react-template" />;
}

export default App;
