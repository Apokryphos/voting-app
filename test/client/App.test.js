import App from 'App';
import React from 'react';
import ReactTestRenderer from 'react-test-renderer';

describe('App rendering', () => {
  it('Renders OK', () => {
    const component = ReactTestRenderer.create(<App />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
