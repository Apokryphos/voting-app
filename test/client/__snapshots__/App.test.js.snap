// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering Renders OK 1`] = `
<h1>
  react-template
</h1>
`;
